src/html/popup.html

```diff
@@ -220,6 +235,60 @@ <h5 id="addTemplateTitle">Add Default or Custom Template</h5>
             </ul>
         </main>
 
+        <!-- Manage custom domains section -->
+        <div class="container">
+            <div id = "customDomainList">
+                <!-- Back and clear buttons -->
+                <br>
+                <div class="col s4 offset-s4">
+                    <div class="row valign-wrapper">
+                            <div class="col s2">
+                                <a id="backInternal" class="btn-floating btn-Medium btn-jti tooltipped center-align" data-position="right" data-delay="50" data-tooltip="Back">
+                                    <i class="material-icons">arrow_back</i>
+                                </a>
+                            </div>
+
+                            <div class="col s8 pull-s1">
```

Comment:
Without it, even with the class center-align, the text appears to be very far right of the center.